test/ProductConstructionService.ScenarioTests/CodeFlowScenarioTestBase.cs

```diff
@@ -69,31 +54,16 @@ protected async Task CheckBackwardFlowGitHubPullRequest(
         Octokit.PullRequest pullRequest = await WaitForPullRequestAsync(targetRepoName, targetBranch);
         IReadOnlyList<Octokit.PullRequestFile> files = await GitHubApi.PullRequest.Files(TestParameters.GitHubTestOrg, targetRepoName, pullRequest.Number);
 
-        try
-        {
-            var versionDetailsFile = files.FirstOrDefault(file => file.FileName == "eng/Version.Details.xml");
-            versionDetailsFile.Should().NotBeNull();
-            versionDetailsFile!.Patch.Should().Contain(GetExpectedCodeFlowDependencyVersionEntry(sourceRepoName, commitSha));
-
-            // Verify new files are in the PR
-            foreach (var testFile in testFiles)
-            {
-                var newFile = files.FirstOrDefault(file => file.FileName == $"{testFile}");
-                newFile.Should().NotBeNull();
-                newFile!.Patch.Should().Be(testFilePatches[testFile]);
-            }
-        }
-        finally
+        var versionDetailsFile = files.FirstOrDefault(file => file.FileName == "eng/Version.Details.xml");
+        versionDetailsFile.Should().NotBeNull();
+        versionDetailsFile!.Patch.Should().Contain(GetExpectedCodeFlowDependencyVersionEntry(sourceRepoName, commitSha));
+
+        // Verify new files are in the PR
+        foreach (var testFile in testFiles)
         {
-            // close the PR
-            await GitHubApi.PullRequest.Update(
-                TestParameters.GitHubTestOrg,
-                targetRepoName,
-                pullRequest.Number,
-                new Octokit.PullRequestUpdate
-                {
-                    State = Octokit.ItemState.Closed
-                });
+            var newFile = files.FirstOrDefault(file => file.FileName == $"{testFile}");
```

Comment:
How does the PR get closed if we removed it here?

Reply:
Or does it get merged instead?

Reply:
we delete the target branch, so the PR gets closed automatically